src/hooks/screen/useChatScreenHook.ts

```diff
@@ -246,14 +245,15 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
         }
       });
     } catch (e) {
-      if (e?.response?.data?.status === 'Channel is blocked') return;
-
-      setTimeout(async () => {
-        await sendChat(message, attachments, iteration + 1, currentChatSchema);
-      }, 1000);
+      console.log('[ERROR] error sending chat', e);
+      // if (e?.response?.data?.status === 'Channel is blocked') return;
     }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in the catch block of the try-catch statement has been simplified. The check for 'Channel is blocked' status and the retry logic have been removed. Instead, an error message is logged to the console.
```diff
- if (e?.response?.data?.status === 'Channel is blocked') return;
+ console.log('[ERROR] error sending chat', e);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->